events/nats.go

```diff
@@ -286,8 +271,11 @@ func (n *NatsJetstream) subscribeAsPull(_ context.Context) error {
 	}
 
 	for _, subject := range n.parameters.Consumer.SubscribeSubjects {
-		subscription, err := n.jsctx.PullSubscribe(subject, n.parameters.AppName, nats.BindStream(n.parameters.Stream.Name))
+		subscription, err := n.jsctx.PullSubscribe(subject, n.parameters.Consumer.Name,
```

Comment:
This should be set to `n.parameters.AppName`, since multiple applications can bind to a single consumer, unless you had other ideas here.